tools/biohansel_bionumeric_converter/bionumeric_convert.xml

```diff
@@ -0,0 +1,40 @@
+<tool id="bionumeric_convert" name="Biohansel Bionumeric Converter" version="0.1.0">
+    <requirements>
+        <requirement type="package" version="0.24.1">pandas</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        python $__tool_directory__/bionumeric_converter.py -f '$Input' -o '$output'
```

Comment:
remove python call. Since you added the [shebang](https://en.wikipedia.org/wiki/Shebang_(Unix)) in previous comment below.